core/src/main/scala/kanaloa/reactive/dispatcher/Regulator.scala

```diff
@@ -118,14 +118,20 @@ object Regulator {
     weightOfLatestMetric:   Double         = 0.5
   )
 
+  private val MaximumEstimatedDelayFactor = 100000
```

Comment:
This is a factor, not bound to a time unit. If the `referenceDelay` is 1 second the `maximumEstimatedDelay` would be `referenceDelay` * `MaximumEstimatedDelayFactor` =   1000000 second.